modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -613,10 +613,10 @@ test.describe('Manage object definitions through View Object Definitions', () =>
 
 		await viewObjectDefinitionsPage.goto();
 
-		await page.locator('.dnd-td.item-actions').first().waitFor();
+		await page.locator('.cell-select-item').first().waitFor();
```

Comment:
@larissacribeiro I added BPM test update commits from https://github.com/liferay-bpm/liferay-portal/pull/3927 on top of the PR, along with minor update on CSS class here. Can you please check if further updates are needed?

cc @ethib137

Reply:
Hey @markocikos, thanks for that! 
Would you mind running the objects suite on this pr? it's ci:test:object. There was a group of tests that were failing due to the [bug](https://liferay.atlassian.net/browse/LPD-43223) Evan mentioned above. 
As I said [here](https://github.com/liferay-bpm/liferay-portal/pull/3913#issuecomment-2511107619), I tried to anticipate the fix of the locators, but it was hard since the table wasn't being rendered, so maybe those remaining failing testscases will still need adjustments on their paths.